### DISCUSSION TITLE
Select options are not readable

### DISCUSSION THREAD

**Discussion by Amorim33** [scalar/scalar#1125] [Q&A]
## Description

When using the light theme, the options for the request client picker are not readable.

![image](https://github.com/scalar/scalar/assets/42624869/7e35a01d-ea06-4ea5-a793-a602d9197bf5)

## Expected behavior

Options text colors should contrast with the background.

### Dark theme example
![image](https://github.com/scalar/scalar/assets/42624869/1b424ff3-e9a4-4f43-8f79-8cc3ba9375cf)

## Theme

```ts
theme: 'default'
```

### Package version
```json
{
    "@scalar/hono-api-reference": "^0.3.67",
}
```

**Comment by Amorim33** [comment]
If this aligns with the library goals, I will be stoked to fix it!😄

**Reply by marclave** [reply]
@Amorim33 this definitely aligns with the libraries goal! feel free to take a swing at this ✨

**Reply by cameronrohani** [reply]
@Amorim33 thanks for the report and tackling this!

My best guess is that since we're adding the `dark-mode` class to that code block it takes the `color-scheme: dark;` css attribute despite the theme being in `light-mode`. 

Would probably need to do something like 

```
.light-mode .dark-mode { 
  color-scheme: light;
}
```

**Reply by Amorim33** [reply]
@cameronrohani I do not know if it is the best approach. However, I solved the bug by adding the following piece of CSS to the `default.css` preset file.
```css
.light-mode .dark-mode {
  color-scheme: dark;
}
```

I guess the light theme's `default-theme-color` was overriding the dark-theme's variables.

Not sure about the reason yet, I will research it a bit more.

![image](https://github.com/scalar/scalar/assets/42624869/4aa3e6af-d101-4e32-83d8-890a0c9cadd4)
